### PR TITLE
 Add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by hamid-murambiwa**
This is a microverse project where I was required to create the form section for the mobile-version of the Portfolio web page.
* Add index.html file
* Add *.css file
*  Add *.svg files
* Add README.md

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -457,5 +457,69 @@ <h3>
...
+    <h4>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h4>
+</div>
+
+<form id="form-grid" action="https://formspree.io/f/xvodvrjv"
+method="POST">
+    <input type="text" placeholder="Full name" name="full-name" maxlength="30" required>
+    <input type="email" placeholder="Email address" name="email" required>
+    <textarea placeholder="Enter text here" name="comment-section" maxlength="500" required>Enter text here</textarea>
```
Comment (DanteAlonsoHT) [inline index.html]: Please, remove the default text "Enter text here" and add it as a placeholder value for your `textarea` element, for instance:
```
placeholder="Enter text here"
```

- [ ] Remove default text in your textarea, and add this default text as placeholder value.

File: index.html [hunk 2]
```diff
@@ -457,5 +457,69 @@ <h3>
...
+    <div class="B-2">
+        <button type="submit" name="comment-section">
+           Get in touch
+        </button>
```
Comment (DanteAlonsoHT) [inline index.html]: - [ ] Add the behavior design according to the Figma design you selected. 
- [ ] [OPTIONAL] Add the next CSS style `cursor: pointer;` for your buttons (when you hover them) to improve the UX.

image

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @hamid-murambiwa and @basnetlaxmi, 

Amazing job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Check the License, you should add your own License without using the License provided by Microverse (with your name)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Murodjon000 — APPROVED** [review]
**Hi Hamid Murambiwa 👋,** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**